config/services.yaml

```diff
@@ -47,6 +47,14 @@ services:
 
     OndraM\CiDetector\CiDetector: null
 
+    Rector\Core\PHPStan\Reflection\TypeToCallReflectionResolver\TypeToCallReflectionResolverRegistry:
+        arguments:
+            -
+                - '@Rector\Core\PHPStan\Reflection\TypeToCallReflectionResolver\ClosureTypeToCallReflectionResolver'
+                - '@Rector\Core\PHPStan\Reflection\TypeToCallReflectionResolver\ConstantArrayTypeToCallReflectionResolver'
+                - '@Rector\Core\PHPStan\Reflection\TypeToCallReflectionResolver\ConstantStringTypeToCallReflectionResolver'
+                - '@Rector\Core\PHPStan\Reflection\TypeToCallReflectionResolver\ObjectTypeToCallReflectionResolver'
+
```

Comment:
These can be removed, it's handled by array autowiring in PHP code

Reply:
Done.